docs/development/design/new-sphinx-guides.rst

```diff
@@ -0,0 +1,186 @@
+         (``python -m http.server``)
+      -  Open http://localhost:8000 in a web browser, and see the HTML
+         documentation (yay!)
+
+   4. Making a change and regenerating the documentation
+
+      -  Add a sentence to ``index.rst`` (no need to explain in detail
+         what reST is at this stage)
+      -  Rebuild HTML ``sphinx-build -W -b html ...``
+      -  Observe that the HTML docs have changed (yay!) (we will have to
+         do this every time we change something)
+
+   5. Simplify documentation building by using Make
+
+      -  Install Make (and its Windows counterpart)
+      -  Make another trivial change
+      -  Build HTML doing ``cd doc && make html``
+      -  Observe that the HTML docs have changed (yay!)
+
+   6. Converting our documentation to PDF
+
+      -  Install TeX Live to create PDF output (should work on every
+         operative system)
+      -  Build LaTeX using ``make latexpdf``
+      -  See that the PDF appeared (yay!)
+
+   7. Appendix: PDF without LaTeX using rinoh (beta)
+
+3. Customizing Sphinx configuration
+
+   1. Changing the HTML theme
+
+      -  Install https://pypi.org/project/furo/
+      -  Change the ``html_theme`` in ``conf.py``
+      -  Rebuild the HTML documentation and observe that the theme has
+         changed (yay!)
+
+   2. Changing the PDF appearance
+
+      -  Add a ``latex_theme`` and set it to ``howto``
+      -  Rebuild ``make latexpdf``
+      -  Check that the appearance changed (yay!)
+
+   3. Enable an extension
+
+      -  Add a string to the ``extensions`` list in ``conf.py`` for
+         ``sphinx.ext.duration``
+      -  Rebuild the HTML docs ``make html`` and notice that now the
+         times are printed (yay!)
+
+4. Writing narrative documentation with Sphinx
+
+   -  First focus on ``index.rst``, gently introducing the learner to
+      reST and mentioning Semantic Line Breaks.
+   -  Then add another ``.rst`` file to teach how ``toctree`` works.
+   -  Then enable Markdown adding ``myst_parser`` to the extensions and
+      authoring a Markdown file, to show that both can exist at the same
+      time.
+   -  Then continue introducing elements of the syntax to add pictures,
+      cross-references, and the like.
+
+5. Using Jupyter notebooks inside Sphinx
+6. Describing code in Sphinx
+
+   -  Explain the Python domain as part of narrative documentation to
+      interleave code with text, include doctests, and justify the
+      usefulness of the next section.
+
+7. Autogenerating documentation from code in Sphinx
+
+.. note 
+
+   - Looks like MathJax is enabled by default now? Can't see a reference in the docstrings
+   - The fact that the ``index.rst`` created by ``sphinx-quickstart`` is written in
+     reStructuredText gets in the way of teaching Markdown (MyST).
+     On the other hand, users need to know reST anyway to write Python docstrings,
+     so it is necessary to teach both.
+
+How-to Guides
```

Comment:
I don't quite understand what this section is meant to include? Is this just additional ideas for our existing guide section? 

https://docs.readthedocs.io/en/latest/guides/

Reply:
Exactly, they are additional ideas to what we have in https://docs.readthedocs.io/en/latest/guides/tools.html. Our guides there are "intermediate level" already I think, the ones I presented are probably more basic.

Reply:
Rephrased this as ideas for new how-to guides covering use cases detected during the interviews.